libcontainer/process_linux.go

```diff
@@ -141,6 +141,16 @@ func (p *setnsProcess) execSetns() error {
 		p.cmd.Wait()
 		return newSystemErrorWithCause(err, "reading pid from init pipe")
 	}
+
+	// Clean up the zombie parent process
+	firstChildProcess, err := os.FindProcess(pid.PidFirstChild)
+	if err != nil {
```

Comment:
> Also are we sure it's not possible to get `PidFirstChild = -1?`

Pretty sure it is impossible since it'd mean the grandchild'd pid has been received. This implies that the child has already been created.

I can just change the nsenter function to differentiate between `child` and `grandchild`.